modules/apps/portal-search-solr8/portal-search-solr8-impl/src/test/java/com/liferay/portal/search/solr8/internal/search/engine/adapter/search/SearchRequestExecutorFixture.java

```diff
@@ -313,6 +323,8 @@ protected void setSolrClientManager(SolrClientManager solrClientManager) {
 		_solrClientManager = solrClientManager;
 	}
 
+	private static BaseSolrQueryAssemblerImpl _baseSolrQueryAssemblerImpl;
```

Comment:
@W2jx If this _baseSolrQueryAssemblerImpl is going to be assigned to different value for each test, then it should not be "static", if it is non-static and you do not need to clean it in tear down either.